### PR TITLE
Add and Remove Info barriers parameters

### PR DESCRIPTION
**Pull request by nibandyo**
Adding new parameters 'AddInformationSegment' and 'RemoveInformationSegment' to support new compliance feature - Information barriers.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -132,6 +132,21 @@ Set-SPOSite -Identity https://contoso.sharepoint.com/sites/site1 -SharingDomainR
 
 Example 7 sets the Sharing Capability to allow external users who accept sharing invitations and sign in as authenticated users, and then specifies an email domain that is allowed for sharing with the external collaborators.
 
+### -----------------------EXAMPLE 8-----------------------------
+
+```powershell
+Set-SPOSite -Identity [https://contoso.sharepoint.com/sites/research] -AddInformationSegment [a17efb47-e3c9-4d85-a188-1cd59c83de32]
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:138]:
```suggestion
Set-SPOSite -Identity https://contoso.sharepoint.com/sites/research -AddInformationSegment a17efb47-e3c9-4d85-a188-1cd59c83de32
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 2]
```diff
@@ -850,6 +866,30 @@ Default value: WebPreviewableFiles
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+### -AddInformationSegment
+
+This parameter allows you to add segment to a SharePoint site.This parameter is only applicable for tenants who have enabled M365 Information barriers capability. Please read (https://<kaarin to provide the link ) documentation to understand Information barriers with SharePoint Online.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:871]: _Please read_ url link is missing

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 3]
```diff
@@ -132,6 +132,21 @@ Set-SPOSite -Identity https://contoso.sharepoint.com/sites/site1 -SharingDomainR
...
+```powershell
+Set-SPOSite -Identity [https://contoso.sharepoint.com/sites/research] -AddInformationSegment [a17efb47-e3c9-4d85-a188-1cd59c83de32]
+```
+
+This example adds InformationSegment 'a17efb47-e3c9-4d85-a188-1cd59c83de32' to the site. 
+
+### -----------------------EXAMPLE 9-----------------------------
+
+```powershell
+Set-SPOSite -Identity [https://contoso.sharepoint.com/sites/research] -RemoveInformationSegment [a17efb47-e3c9-4d85-a188-1cd59c83de32]
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:146]:
```suggestion
Set-SPOSite -Identity https://contoso.sharepoint.com/sites/research -RemoveInformationSegment a17efb47-e3c9-4d85-a188-1cd59c83de32
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md (hunk 2 above)
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:871]:
```suggestion
This parameter allows you to add a segment to a SharePoint site. This parameter is only applicable for tenants who have enabled Microsoft 365 Information barriers capability. Please read (https://<kaarin to provide the link ) documentation to understand Information barriers with SharePoint Online.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 4]
```diff
@@ -850,6 +866,30 @@ Default value: WebPreviewableFiles
...
+
+```yaml
+Type: GUID
+Required: False
+Position: Named
+Default value: None
+```
+
+### -RemoveInformationSegment
+This parameter allows you to remove segment from a SharePoint site.This parameter is only applicable for tenants who have enabled M365 Information barriers capability. Please read (https://<kaarin to provide the link ) documentation to understand Information barriers with SharePoint Online.
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:883]:
```suggestion
This parameter allows you to remove a segment from a SharePoint site. This parameter is only applicable for tenants who have enabled Microsoft 365 Information barriers capability. Please read (https://<kaarin to provide the link ) documentation to understand Information barriers with SharePoint Online.
```
The link should be added here too.

**Comment by kenwith** [comment]
@kaarins see note above on this one too.. @nibandyo this will go live, please update the link ASAP.

**Pull request merged**